pkg/kev/init.go

```diff
@@ -54,44 +65,56 @@ func (r *InitRunner) Run() (WritableResults, error) {
 	}
 
 	r.UI.Header("Detecting Skaffold settings...")
-	if r.config.skaffold {
+	if r.config.Skaffold {
 		if skManifest, err = r.CreateOrUpdateSkaffoldManifest(); err != nil {
 			return nil, err
 		}
 	} else {
 		r.UI.Output("Skipping - no Skaffold options detected")
 	}
 
-	return createInitWritableResults(r.workingDir, r.manifest, skManifest), nil
+	return createInitWritableResults(r.WorkingDir, r.manifest, skManifest), nil
 }
 
 // EnsureFirstInit ensures the project has not been already initialised
 func (r *InitRunner) EnsureFirstInit() error {
+	if err := r.eventHandler(PreEnsureFirstInit, r); err != nil {
+		return errors.Errorf("%s\nwhen handling fired event: %s", err.Error(), PreEnsureFirstInit)
```

Comment:
And here as well, unless there is a reason why we shouldn't be more explicit? This comment applies to other places where we return error like this.